Packs/QutteraWebsiteMalwareScanner/Integrations/QutteraWebsiteMalwareScanner/QutteraWebsiteMalwareScanner.py

```diff
@@ -0,0 +1,208 @@
+        readable_output = f'Failed to scan domain {domain}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="QutteraWebsiteMalwareScanning.Start",
+        outputs_key_field="error",
+        outputs=start
+    )
+
+
+def rescan_status(client: Client, args: dict) -> CommandResults:
+    """
+    Retrieves scan status for previously submitted domain
+    name or URL.
+
+    """
+    domain = args.get("domain")
+    if not domain:
+        raise ValueError('domain is missing')
+
+    rescan = client._api_request(domain=domain, request_type="PUT", operation="rescan")
+    readable_output = f'Rescan of {domain}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="QutteraWebsiteMalwareScanning.Status",
+        outputs_key_field="error",
+        outputs=rescan
+    )
+
+
+def scan_report(client: Client, args: dict) -> CommandResults:
+    """
+    Retrieves scan report for previously submitted domain
+    or URL in context or as a File (default)
+
+    """
+    domain = args.get("domain")
+    if not domain:
+        raise ValueError('domain is missing')
+
+    report = client._api_request(domain=domain, request_type="GET", operation="statscan")
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.report",
+        outputs_key_field="error",
+        outputs=report
+    )
+
+
+def report_malware(client: Client, args: dict) -> CommandResults:
+    """
+    Returns malware status for given domain
+
+    """
+    domain = args.get("domain")
+    if not domain:
+        raise ValueError('domain is missing')
+
+    malware = client._api_request(domain=domain, request_type="GET", operation="malware")
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.malware",
+        outputs_key_field="error",
+        outputs=malware
+    )
+
+
+def status_blacklist(client: Client, args: dict) -> CommandResults:
+    """
+    Returns url blacklisting information and reputation.
+
+    """
+    domain = args.get("domain")
+    if not domain:
+        raise ValueError('domain is missing')
+
+    start = client._api_request(domain=domain, request_type="GET", operation="blacklist")
+    readable_output = f"Blacklisted URI's of {domain}"
+    if start.get('errorstr') != 'success':
+        readable_output = f'Failed to get the report for domain {domain}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="",
+        outputs_key_field='QutteraWebsiteMalwareScanning.domain',
+        outputs=start
+    )
+
+
+def test_module(client: Client):
+    """Performs basic request to ensure API server is running
+
+    Return True if request succeeded
+    """
+    resp = client._test_module()
+    if resp.get('errorstr') == 'success':
+        return 'ok'
+    raise DemistoException(f'Service is down: ${resp}')
+
+
+def main()  # pragma: no cover
```

Comment:
```suggestion
def main():  # pragma: no cover
```

My bad, missed the ":"